protocols/enocean/org.eclipse.smarthome.protocols.enocean.basedriver.impl/enocean_config/enocean_config.txt

```diff
@@ -0,0 +1,8 @@
+#Wed Dec 17 17:11:53 CET 2014
+#0x00112233_RORG_FUNC_TYPE_FRIENDLYNAME_DESCRIPTION=aa#bb#cc##dd
```

Comment:
Hi Kai,
I just update the thing.
Antonin.

De : Kai Kreuzer [mailto:notifications@github.com]
Envoyé : jeudi 5 mars 2015 17:33
À : eclipse/smarthome
Cc : CHAZALET Antonin OAB
Objet : Re: [smarthome] EnOcean OSGi Base Driver (https://bugs.eclipse.org/bugs/show_bug.cgi?id=457338) (#155)


In protocols/enocean/org.eclipse.smarthome.protocols.enocean.basedriver.impl/enocean_config/enocean_config.txt<https://github.com/eclipse/smarthome/pull/155#discussion_r25877572>:

> @@ -0,0 +1,8 @@

> +#Wed Dec 17 17:11:53 CET 2014

> +#0x00112233_RORG_FUNC_TYPE_FRIENDLYNAME_DESCRIPTION=aa#bb#cc##dd

Is it ok for ESH?

I just do not think it is ok for an official distributable, which is meant to be included in products. If it is required by the test/sample client, it should imho a part of this test scenario, e.g. offered as a fragment that can be added for this purpose, don't you think?

—
Reply to this email directly or view it on GitHub<https://github.com/eclipse/smarthome/pull/155/files#r25877572>.

_________________________________________________________________________________________________________________________

Ce message et ses pieces jointes peuvent contenir des informations confidentielles ou privilegiees et ne doivent donc
pas etre diffuses, exploites ou copies sans autorisation. Si vous avez recu ce message par erreur, veuillez le signaler
a l'expediteur et le detruire ainsi que les pieces jointes. Les messages electroniques etant susceptibles d'alteration,
Orange decline toute responsabilite si ce message a ete altere, deforme ou falsifie. Merci.

This message and its attachments may contain confidential or privileged information that may be protected by law;
they should not be distributed, used or copied without authorisation.
If you have received this email in error, please notify the sender and delete this message and its attachments.
As emails may be altered, Orange is not liable for messages that have been modified, changed or falsified.
Thank you.